packages/babel-core/src/config/full.js

```diff
@@ -223,6 +226,7 @@ const loadDescriptor = makeWeakCache(function* (
       ...context,
       ...makeAPI(cache),
     };
+    api.addExternalDependency = fileName => dependencies.add(fileName);
```

Comment:
Can you move `addExternalDependency` to `packages/babel-core/src/config/helpers/config-api.js`? We should also add it to the `PluginAPI` interface.

Reply:
Fixed.